ipt_connect/IPTdev/tactics.py

```diff
@@ -1,8 +1,3 @@
-from django.shortcuts import render
-from django.views.decorators.cache import cache_page
-from models import *
-from django.contrib.auth.decorators import user_passes_test
-import parameters as params
```

Comment:
I'd prefer to have `params` available in all the files as well as all the models. Are you sure that tactics works without importing models?